grails-app/services/org/pih/warehouse/report/NotificationService.groovy

```diff
@@ -133,7 +133,7 @@ class NotificationService {
         def recipientItems = shipmentInstance.shipmentItems.groupBy {it.recipient }
         recipientItems.each { Person recipient, items ->
             if (emailValidator.isValid(recipient?.email)) {
-                def subject = g.message(code: "email.yourItemShipped.message", args: [shipmentInstance.shipmentNumber])
+                def subject = g.message(code: "email.yourItemShipped.message", args: [shipmentInstance.origin, shipmentInstance.destination, shipmentInstance.shipmentNumber])
```

Comment:
We should ask Kelsey if this should be the location.locationNumber or location.name

Reply:
In example in the ticket there is "Please be informed that an item you requested has arrived at Miami Warehouse: Haiti Stock." so I assume that is should be name.

Reply:
Ok, but we still need to reference the name propery explicitly in case the toString() ever changes.

Reply:
@jmiranda Done.